build.sbt

```diff
@@ -63,7 +63,7 @@ lazy val core = crossProject(JSPlatform, JVMPlatform, NativePlatform)
   .jvmSettings(commonJvmSettings)
   .platformsSettings(JVMPlatform, JSPlatform)(
     libraryDependencies ++= Seq(
-      "org.scalatest" %%% "scalatest" % "3.0.6-SNAP5" % Test,
+      "org.scalatest" %%% "scalatest" % "3.0.8" % Test,
```

Comment:
You need to update to `3.1.0-SNAP13` (which will likely require some changes to the usage code), 3.1.0 is used by `discipline-scalatest` and is not compatible with 3.0.8

Reply:
Yes, it works! Thank you!